.eslintrc.json

```diff
@@ -1,14 +1,11 @@
 {
 	"root": true,
-	"extends": "5app",
-	"plugins": [
-		"promise",
-		"security"
-	],
+	"extends": ["5app", "prettier"],
+	"plugins": ["promise", "security", "prettier"],
```

Comment:
I am wondering if we still need to keep "promise" here as this PR is targeting recent versions of JS/Node and it is already relying on recent versions of ESLint config 🤔